cli/template/addons/tailwind/tailwind.config.cjs

```diff
@@ -2,7 +2,21 @@
 module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
```

Comment:
Not sure if we want to ship this, seems like something everyone would rip out. Are normal TW colors not "close-enligt"?

Reply:
the light ones are fine, but `purple-900` etc dont go dark enough. could just hardcode some colors inline.

Reply:
> the light ones are fine, but `purple-900` etc dont go dark enough. could just do `bg-[#abcdef]` etc where necessary.

I would prefer this yeah. We don't wanna configure anything.

Reply:
using all stock colors now except for the background (darker than purple-900)